index.d.ts

```diff
@@ -0,0 +1,104 @@
+// TypeScript Version: 3.7
+
+import {Test} from 'unist-util-is'
+import * as unist from 'unist'
+
+export {}
```

Comment:
Empty export statement. Please remove this line

Reply:
the empty export statement is there so that explicit exporting is turned on. otherwise the Null type would be exported

Reply:
This is CJS module. Only namespace and the function will be exported. 

How `Null` can be exported?

Reply:
<img width="885" alt="Screen Shot 2020-01-19 at 0 17 00" src="https://user-images.githubusercontent.com/5865853/72675389-0e021180-3a51-11ea-8194-2570291124de.png">

As you can see in the screenshot, `Null` is not accessible unless it is defined in the namespace.

Reply:
Odd... this was something dtslint asked for. I'll remove it and we'll see what happens.